en/drafts/originals/facial-recognition-ai-python.md

```diff
@@ -415,37 +425,47 @@ data.to_csv('YearbookOutput.csv', index=False)
 
-If we might draw some qualitative conclusions from this small sample dataset, 1961 yearbook is the first yearbook where smiles begin to rival non-smiles in total numbers. Claudia Goldin and Lawrence F. Katz have argued that while American higher education gradually became more and more coed from the mid-1800s to the mid-1900s, the switch from single-sex to coed education accelerated rapidly in the 1960s and 1970s.[^9] The greater share of smiling faces in yearbooks from the 1960s might be part and parcel with other, wider social changes in the American collegiate landscape. Further analysis of this data by sorting faces based on gender would be needed to confirm whether Bethel's student body became markedly more female in these years, but of course that process would be fraught by the ethical challenges of gender-based facial recognition [described above](#ethical-issues-with-facial-recognition).
+If we draw some qualitative conclusions from this small sample dataset, 1961 is the first year where smiles began to rival non-smiles in total numbers. Claudia Goldin and Lawrence F. Katz have argued that while American higher education gradually became more and more coed (mixed genders) from the mid-1800s to the mid-1900s, the switch from single-sex to coed education accelerated rapidly in the 1960s and 1970s.[^9] The greater share of smiling faces in yearbooks from the 1960s might be a part of other, wider social changes in the American collegiate landscape. Further analysis of this data by sorting faces based on gender would be needed to confirm whether Bethel's student body became markedly more female in these years, but of course that process would be fraught by the ethical challenges of gender-based facial recognition [described above](#ethical-issues-with-facial-recognition).
 
-Let's continue our analysis and zero in on the "Error Weight" column of the `.csv` file:
+Let's continue our analysis and zero in on the **Error Weight** column of the `.csv` file:
 
```

Comment:
As part of our commitment to keeping our pages as light and accessible as possible, we prefer to convert images of spreadsheets to markdown tables instead.